build_scripts/windows/scripts/build.cmd

```diff
@@ -43,6 +48,7 @@ set ARTIFACTS_DIR=%~dp0..\artifacts
 mkdir %ARTIFACTS_DIR%
 set TEMP_SCRATCH_FOLDER=%ARTIFACTS_DIR%\cli_scratch
 set BUILDING_DIR=%ARTIFACTS_DIR%\cli
+set ZIP_ROOT_FOLDER="AzureCLI"
```

Comment:
This will be the first time we introduce the term `AzureCLI`. Other alternatives can be `azure-cli`, `az`. 

However, I downloaded several other Microsoft-owned apps, like

- PowerShell: https://github.com/PowerShell/PowerShell/releases/download/v7.4.1/PowerShell-7.4.1-win-x64.zip from https://github.com/PowerShell/PowerShell/releases/tag/v7.4.1
- VS Code: https://code.visualstudio.com/sha/download?build=stable&os=win32-x64-archive from https://code.visualstudio.com/Download.
- PsTools: https://download.sysinternals.com/files/PSTools.zip from https://learn.microsoft.com/en-us/sysinternals/downloads/pstools

All put their content in the root folder.